TESTING.md

```diff
@@ -0,0 +1,59 @@
+# Testing CP-Ansible
+
+Each role has Molecule configuration which enable fast and easy development and testing with Docker containers.
+
+## Prerequisites
+
+Make sure Docker is running and increase the CPU/Memory available to Docker in your Preferences. Next install these python libraries:
```

Comment:
I would do this a list of items:  "The following are required for Molecule deployment and testing:

1. Docker 
2. .... 

I would also link to the official install documentation for each component.